### PR TITLE
mailmap update and regenerate AUTHORS

### PR DESCRIPTION
**Pull request by neersighted**
Update the mailmap with my contact info, and regenerate the AUTHORS file

### CODE REVIEW

**Review by thaJeztah — APPROVED** [review]
LGTM

File: AUTHORS [hunk 1]
```diff
@@ -433,6 +434,7 @@ Cristian Staretu <cristian.staretu@gmail.com>
 cristiano balducci <cristiano.balducci@gmail.com>
 Cristina Yenyxe Gonzalez Garcia <cristina.yenyxe@gmail.com>
 Cruceru Calin-Cristian <crucerucalincristian@gmail.com>
+cui fliter <imcusg@gmail.com>
 CUI Wei <ghostplant@qq.com>
 cuishuang <imcusg@gmail.com>
```
Comment (tianon) [inline AUTHORS:439]: :see_no_evil:

**Review by tianon — APPROVED** [review]

File: AUTHORS (hunk 1 above)
Comment (neersighted) [reply AUTHORS:439]: Good eye

Comment (thaJeztah) [reply AUTHORS:439]: oh! nice one. usually I'm spotting these. Did actually check git log for one of the other ones to see if we had a "correct" name 😂

Comment (tianon) [reply AUTHORS:439]: Nope, just noticed it in the diff while reviewing in the web UI -- I imagine there could be others, but I was not thorough :sweat_smile:

**Comment by tianon** [comment]
Let's merge as-is and look for improvement as a follow-up. :+1:

**Pull request merged**